webclient/components/papers/note-view/templates/zl4-content-during-appt-of-a-single-ct.vue

```diff
@@ -64,9 +64,9 @@
          Each appt gets a slide of its own         -->
 
       <ul class="hs full no-scrollbar" id="container-for-all-appointments">
-        <section v-for="item in cf_get_entity_value_during_each_appt" :key="item.id">
-          <li class="item" id="container-for-one-appointment">
-            <div v-if="currentApptObj.apptStartMilliSecsOnCalendar !== item.apptStartMilliSecsOnCalendar">
+        <section v-for="(item, itemIndex) in cf_get_entity_value_during_each_appt" :key="itemIndex">
```

Comment:
item and itemindex names could have been better.

Reply:
rename the variables